foundry.toml

```diff
@@ -1,4 +1,5 @@
 [default]
+out = "artifacts"
```

Comment:
Moves default `/out/` directory to `/artifacts/` so importing the artifacts in tests becomes more explicit.